sdk/identity/azure-identity/azure/identity/aio/_internal/token_binding_transport.py

```diff
@@ -0,0 +1,38 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""
+Transport class for the asynchronous WorkloadIdentityCredential with token proxy support.
+"""
+import ssl
+from typing import Any
+
+from azure.core.pipeline.transport import (  # pylint: disable=non-abstract-transport-import, no-name-in-module
+    AioHttpTransport,
+)
+from azure.core.rest import HttpRequest
+
+from ..._internal.token_binding_transport_mixin import TokenBindingTransportMixin
+
+
+class CustomAioHttpTransport(TokenBindingTransportMixin, AioHttpTransport):
```

Comment:
As we discussed, do we plan to allow user to choose? e.g. trio instead of aiohttp? or httpx?

Reply:
The concern is this creates a hard dependency on aiohttp.